nemo/collections/llm/adapter/lora.py

```diff
@@ -0,0 +1,65 @@
+from dataclasses import dataclass, field
+from typing import List, Literal
+
+from megatron.core import parallel_state
+from nemo.collections.llm.adapter.base import AdapterWrapper, PEFTConfig
+from nemo.collections.nlp.modules.common.megatron.adapters.parallel_adapters import ParallelLinearAdapter
+
+
+class AdapterParallelAdd(AdapterWrapper):
+    """Example: LoRA Adapter"""
+
+    def forward(self, x):
+        linear_output, bias = self.to_wrap(x)
+        if isinstance(linear_output, tuple) and len(linear_output) == 2:
+            linear_output, layernorm_output = linear_output
+            adapter_output = self.adapter(layernorm_output)
+        else:
+            adapter_output = self.adapter(x)
+        return linear_output + adapter_output, bias
+
+
+@dataclass
+class LoRAConfig(PEFTConfig):
+    target_modules: List[str] = field(
```

Comment:
Wouldn't it make more sense to make it batteries-included by add a sensible default for the target_modules?